AUTHORS

```diff
@@ -0,0 +1,3 @@
+Adir Gabai <adir@stratoscale.com>
+Eran <eran@stratoscale.com>
```

Comment:
just like Madonna :)
@eran-stratoscale do you want me to update it manually or just keep it like this?